src/actix/headers.rs

```diff
@@ -0,0 +1,68 @@
+use crate::event::SpecVersion;
+use actix_web::http::header;
+use actix_web::http::{HeaderName, HeaderValue};
+use lazy_static::lazy_static;
+use std::collections::HashMap;
+use std::str::FromStr;
+
+macro_rules! unwrap_optional_header {
```

Comment:
Not for this PR, but maybe for future refactorings: a lot of these macros are copy paste, because the APIs are practically the same even if their names are different, so it would be nice to find a way to unify them

Reply:
Addressed in #146